packages/tupaia-web/src/features/Map/MapOverlaysLayer/MapOverlaysLayer.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import { LegendProps } from '@tupaia/ui-map-components';
+import { useMapOverlayMapData } from '../utils';
+import { PolygonLayer } from './PolygonLayer';
+import { MarkerLayer } from './MarkerLayer';
+
+export const MapOverlaysLayer = ({
+  hiddenValues,
+}: {
+  hiddenValues: LegendProps['hiddenValues'];
+}) => {
+  const { serieses, measureData, isLoading } = useMapOverlayMapData(hiddenValues);
+  return (
+    <>
+      <PolygonLayer measureData={measureData} serieses={serieses} />
+      {isLoading ? null : <MarkerLayer measureData={measureData} serieses={serieses} />}
```

Comment:
I'm not sure if this will make the markers disappear when changing entity, e.g. when clicking on the marker to go to another entity. Probably fine, but will see if Andrew notices a regression there.